app/cmd/root.go

```diff
@@ -0,0 +1,45 @@
+package cmd
+
+import (
+	"database/sql"
+	"fmt"
+
+	_ "github.com/lib/pq"
+	"github.com/ngavinsir/golangtraining"
+	postgresRepository "github.com/ngavinsir/golangtraining/internal/postgres"
+)
+
+var (
+	paymentCodesRepository golangtraining.IPaymentCodesRepository
+)
+
+const (
+	host     = "localhost"
+	port     = 5432
+	user     = "postgres"
+	password = "password"
+	dbname   = "golangbeginner"
+)
+
+func init() {
+	dbConn := initDB()
+
+	paymentCodesRepository = postgresRepository.NewPaymentCodesRepository(dbConn)
+}
+
+func initDB() (db *sql.DB) {
```

Comment:
(nitpick)
reduce the usage of named returns **unless the data type has conflicts**, e.g.

```go
GetLocation() (float64, float64, error)
```
should be 
```go
GetLocation() (lat, long float64, err error)
```
but
```go
GetLatitude() (lat float64, err error)
```
should be 
```go
GetLatitude() (float64, error)
```

Reply:
ohh okay, TIL, thanks